serializers/src/OBJ/objSerializer.ts

```diff
@@ -67,7 +68,13 @@ export class OBJExport {
             }
 
             for (var i = 0; i < trunkVerts.length; i += 3) {
-                output.push("v " + trunkVerts[i] + " " + trunkVerts[i + 1] + " " + trunkVerts[i + 2]);
+                // Babylon default is left handed, while OBJ default is right handed
+                // By default we serialize in OBJ inverting the X vertices
+                if (sceneUsesRightHandedSystem) {
```

Comment:
I would recommend just asking the scene. The scene can be retrieved from meshes[0].getScene

We do not need an extra parameter

Reply:
nice suggestion, just changed that to make everything much simpler !